utils.py

```diff
@@ -721,6 +722,86 @@ def get_hashable_value(value):
         return value
 
 
+def convert_to_text_angular(html_data):
+    """This fucntion converts the html to text angular supported format.
+
+    Args:
+        html_data: str. HTML string to be converted.
+
+    Returns:
+        str. The converted HTML string.
+    """
+    soup = bs4.BeautifulSoup(html_data, 'html.parser')
+
+    tags_to_remove = ['div', 'span', 'code', 'table', 'td', 'tr', 'tbody']
+
+    for tag_name in tags_to_remove:
+        for tag in soup.findAll(tag_name):
+            tag.unwrap()
+
+    for br in soup.findAll('br'):
```

Comment:
Please add comments explaining what each block is doing here and below. E.g., `Ensure that every line break is a child of <b>, <i>, <li>, or <p>`.

Reply:
Done!